### PR TITLE
Feature/home

### PR DESCRIPTION
**Pull request by kinhkyo**

### CODE REVIEW

File: src/components/TSAutocomplete/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        placeholder="City, province or region"
+        filterOption={(inputValue, option) => option.props.children.toUpperCase().indexOf(inputValue.toUpperCase()) !== -1}
+        {...props}
+      />
+    );
+  }
+}
+
+TSAutocomplete.propTypes = {
+
```
Comment (luannm) [inline src/components/TSAutocomplete/index.js]: Missing prop types validation

File: src/components/TSAutocomplete/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react';
+import { AutoComplete } from 'antd';
+import PropTypes from 'prop-types';
+import './index.less';
+
+const dataSource = ['TP.Hồ Chí Minh', 'Hà Nội', 'Hải Phòng'];
```
Comment (luannm) [inline src/components/TSAutocomplete/index.js]: Data source should be passed in via props. Not hardcode inside component => Cannot reuse

File: src/components/TSFeatureJobCard/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import IconMaker from '../../assets/images/icon-map-maker.svg';
+import IconLove from '../../assets/images/icon-love.svg';
+
+class TSFeatureJobCard extends PureComponent {
+  render() {
+    const { className, ...props } = this.props;
+
+    return (
+      <Card hoverable {...props} className={classnames('ts-feature-job-card', 'ts-feature-job-card--border-top', className)}>
+        <Row>
```
Comment (luannm) [inline src/components/TSFeatureJobCard/index.js]: Is it component done already? Or just for demo purpose? Cause I see you're set fixed data for component

**Review by luannm — COMMENTED** [review]
Pls check my review @kinhkyo

File: src/components/TSFeatureJobCard/index.js (hunk 3 above)
Comment (kinhkyo) [reply src/components/TSFeatureJobCard/index.js]: All of the components just hardcode data to test UI, I will change it to dynamic tonight. Please wait or you can just use for display on UI.

Comment (luannm) [reply src/components/TSFeatureJobCard/index.js]: okay, I see! then let wait after you've updated it to dynamic and we can merge this PR

**Comment by kinhkyo** [comment]
@luannm Pls help me verify again.

**Pull request merged**